content/security/index.adoc

```diff
@@ -4,48 +4,60 @@ title: "Jenkins Security"
 section: security
 ---
 
+The Jenkins project takes security seriously.
+We make every possible effort to ensure users can adequately secure their automation infrastructure.
```

Comment:
Real text editors allow word wrap.

While it's no big deal for diffs with the right options, searching for the file containing a certain sentence becomes a huge PITA with arbitrary hard line breaks.